app/controllers/home_controller.rb

```diff
@@ -2,5 +2,6 @@ class HomeController < ApplicationController
   after_action :verify_authorized, except: [:index]
 
   def index
+    redirect_to current_user.after_sign_in_path if user_signed_in?
```

Comment:
^^^ This is cool! I think it may be overkill @jedschneider . I would say more often than not, the user will not be redirected and instead have to login, so I'm not inclined to add more complexity for the smaller use case.